impress-org/givewp
# Feature: Add manual donations recommendation

## Description

This pull request introduces a new feature to the Donations table - a "Manual Donations" button/link. When users hover over this element, a pop-up notice appears, providing information about the benefits of the manual donations add-on for those who may not be familiar with it. Clicking on the button/link opens a new window that follows a UTM link specifically designed for the manual donations add-on. Importantly, this button/link is only visible to users who do not have the manual donations feature installed and active.

After speaking with Jeffrey several other minor styling suggestions were included and applied to the other recommendations. These changes include font-size on the"Pro tip" messages, text-alignment on the "recurring recommendation button" while in mobile/menu view, as well as a contextual change from "Pro tip:" -> "PRO TIP:"

## Affects

Donation Table

## Visuals


https://github.com/impress-org/givewp/assets/75056371/00f32e34-b87a-410b-8146-cdd703fd9f69


## Testing Instructions

- Visit the Donations table
- Hover over the enter donations button and view the notice.
- Click the button, it should populate a new window with the appropriate UTM link.
- Activate Manual Donations add-on and revisit the Donations table. The link should be replaced with the regular "New Donation" - Manual donation button for creating manual donations.
 
## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204782112762770
